packages/asc-ui-beta/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@amsterdam/asc-ui-beta",
+  "description": "React UI Components with new styling that developers at de Gemeente Amsterdam can reuse in multiple projects",
+  "license": "MPL-2.0",
+  "author": "Gemeente Amsterdam",
+  "homepage": "https://github.com/Amsterdam/amsterdam-styled-components#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Amsterdam/amsterdam-styled-components.git",
+    "directory": "packages/asc-ui-beta"
+  },
+  "bugs": {
+    "url": "https://github.com/Amsterdam/amsterdam-styled-components/issues"
+  },
+  "version": "0.35.0",
```

Comment:
This version isn't correct, or is it? Do you want to keep it in sync with `asc-ui`?

Reply:
You're right, it should be `0.1.0`